default/stringtables/en.txt

```diff
@@ -339,11 +339,14 @@ PROJECT_BT_STOCKPILE_SHORT_DESC
 Transfers PP into the imperial stockpile
 
 PROJECT_BT_STOCKPILE_DESC
-'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. Anything enqueued below the transfer project at a supply-connected planet will not be allocated PP.
+'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. The amount to transfer and how many times the transfer should repeat can be specified. PP allocated to the transfer are added to the imperial stockpile.
```

Comment:
1st and 3rd sentences seem redundant. Could replace the 1st with the 3rd:

Production points (PP) allocated to the Stockpile Transfer project are transferred to the imperial stockpile. The amount to transfer and how many times the transfer should repeat can be specified.